ionic2/app/components/types/types.component.html

```diff
@@ -1,3 +1,3 @@
-<ion-badge [style.background-color]="poketype.color" *ngFor="let poketype of typeArray">
+<ion-badge [style.background-color]="poketype.color" *ngFor="let poketype of typesArray">
```

Comment:
May I ask what is the reason behind this? We have to change this in different places too. Couldn't it just stay like this?